.gitignore

```diff
@@ -8,4 +8,5 @@ scripts/auto_gpt_workspace/*
 *.mpeg
 .env
 last_run_ai_settings.yaml
-outputs/*
\ No newline at end of file
+outputs/*
+.env.template
```

Comment:
.env.template is in the repo and it isn't supposed to be modified without committing. So this doesn't look right for me

Reply:
I listen advice of reviewer last time xd